maxim/test_runs/test_run_builder.py

```
@@ -684,12 +736,19 @@ def process_dataset_entry(
             ],
         ) -> None:
             try:
-                rowData: ManualData = row.to_dict()["data"]
+                rowData: LocalData = row.to_dict()["data"]
                 if rowData is None:
                     raise ValueError(f"Dataset entry {index} is missing")
```

Ruff (0.11.9)

Abstract `raise` to an inner function

(TRY301)